File: server_implementation/src/main/java/com/example/localserver/DTO/RegisteredUserDTO.java
package com.example.localserver.DTO;

public class RegisteredUserDTO {

    private String uniqueCode;
    private String indexNr;

    public RegisteredUserDTO(){}

    public RegisteredUserDTO(String uniqueCode, String indexNum) {
        this.uniqueCode = uniqueCode;
        this.indexNr = indexNum;
    }

    public String getUniqueCode() {
        return uniqueCode;
    }

    public void setUniqueCode(String uniqueCode) {
        this.uniqueCode = uniqueCode;
    }

    public String getIndexNr() {
        return indexNr;
    }

    public void setIndexNr(String indexNr) {
        this.indexNr = indexNr;
    }

    @Override
    public String toString() {
        return "{" + "uniqueCode='" + uniqueCode + '\'' +
                ",indexNum='" + indexNr + '\'' + '}';
    }
}

File: Server/src/ServerAPI/DTOs/RegisteredUserDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServerAPI.DTOs
{
    public class RegisteredUserDTO
    {
        public RegisteredUserDTO() { }
        public RegisteredUserDTO(string uniqueCode, string indexNr)
        {
            UniqueCode = uniqueCode;
            IndexNr = indexNr;
        }
        public string UniqueCode { get; set; }
        public string IndexNr { get; set; }


        public override string ToString()
        {
            return "{" + "uniqueCode='" + UniqueCode + '\'' +
                 ",indexNum='" + IndexNr + '\'' + '}';
        }
    }
}


